README.md

```diff
@@ -4,30 +4,34 @@ Order files can reduce app startup time by co-locating symbols that are accessed
 file by launching the app in an XCUITest. Read all about how order files work in [our blog post](https://www.emergetools.com/blog/posts/FasterAppStartupOrderFiles).
 
 Setting up your app for order files requires 2 steps:
-1 - Use this library to generate an order file as part of your XCUITest. The library instruments app launch in the UI test and uses the results to generate an optimized order file
-2 - Re-build the app with the order file. Once the order file is generated you build the app again, this time passing the order file as an option to the linker.
 
-## Installation
+1. Generate an order file as part of an XCUITest. FaultOrdering instruments app launch in the UI test and uses the results to generate an optimized order file.
+2. Once the order file is generated, build the app again, this time passing the order file as an option to the linker.
 
-Create a UI testing target using XCUITest. Add the package dependency to your Xcode project using the URL of this repository (https://github.com/getsentry/FaultOrdering).
-Add `FaultOrderingTests` and `FaultOrdering` as a dependency of your new UI test target.
+## Package Installation
 
-### Generating the linkmap
+- Create a UI testing target using XCUITest.
+- Add https://github.com/getsentry/FaultOrdering to your project via Swift Package Manager.
+- Add `FaultOrderingTests` and `FaultOrdering` as dependencies of your new UI test target.
 
-To use this package you'll need to tell Xcode to generate a linkmap for your main app binary. In your app's Xcode target, set the following build settings:
+## Generating the linkmap
```

Comment:
I think this is a different step than installation, so I promoted it to the same header level as `## Installation`. Similar with most of the other headers.